lightly/cli/_checkpoint_cli_simclr.py

```diff
@@ -0,0 +1,49 @@
+""" SimCLR Model """
+
+# Copyright (c) 2020. Lightly AG and its affiliates.
+# All Rights Reserved
+
+import warnings
+
+import torch
+import torch.nn as nn
+
+from lightly.models.modules import SimCLRProjectionHead
+
+
+class _SimCLR(nn.Module):
+    """Implementation of the SimCLR architecture for using the checkpoint
```

Comment:
Just write "Implementation of SimCLR used by the command-line interface." and maybe add "Provides backwards compatability with old checkpoints.".

And please don't forget that punctuation is important 😉